src/app/drivingLicence/controllers/validate.js

```diff
@@ -1,23 +1,24 @@
-const axios = require("axios");
 const BaseController = require("hmpo-form-wizard").Controller;
 
 const {
-  API_BASE_URL,
-  API_CHECK_DRIVING_LICENCE_PATH,
-  API_BUILD_CLIENT_OAUTH_RESPONSE_PATH,
+  API: {
+    PATHS: { CHECK },
+  },
 } = require("../../../lib/config");
 const logger = require("hmpo-logger").get();
 
 class ValidateController extends BaseController {
   async saveValues(req, res, callback) {
+
     const firstName = req.sessionModel.get("firstName");
     const middleNames = req.sessionModel.get("middleNames");
     const forenames =
       middleNames === ""
         ? firstName.split(" ")
         : firstName.split(" ").concat(middleNames.split(" "));
     const attributes = {
-      documentNumber: req.sessionModel.get("documentNumber"),
+      drivingLicenceNumber: req.sessionModel.get("drivingLicenceNumber"),
```

Comment:
Feel free to ignore my comment on the backend PR looks like we're now using drivingLicenceNumber

Reply:
Needed to to make it play nice with the frontend